test/index.test.js

```diff
@@ -0,0 +1,61 @@
+const MemoryDatabase = require('../lib/index');
+
+describe('memory database', () => {
+    let db = null;
+    beforeEach(() => { 
```

Comment:
```suggestion
    beforeEach(() => { 
        db = new MemoryDatabase();
    });
```
Try to keep indentation consistent across files -- helps to catch syntax errors in the future.